pyaedt/edb_core/EDB_Data.py

```diff
@@ -587,6 +589,48 @@ def filling_material_name(self, value):
             self._filling_material_name = value
             self.update_layers()
 
+    @property
+    def negative_layer_value(self):
+        """Negative layer.
+
+        Returns
+        -------
+        bool
+            True if the layer is negative, else False.
+        """
+        try:
+            self._negative_layer = self._layer.GetNegative()
+        except:
+            pass
```

Comment:
Quitely passing errors should be avoided. We should either log this or raise a warning. Also, at the very least use a `pragma: no cover` so that we don't insist on coverage for these lines.